classes/pdf/HTMLTemplate.php

```diff
@@ -136,8 +136,14 @@ public function assignCommonHeaderData()
             $width *= $ratio;
         }
 
+        if ('' !== _MEDIA_SERVER_1_ || _MEDIA_SERVER_2_ || _MEDIA_SERVER_3_) {
+            $logo_path = Tools::getShopProtocol() . Tools::getMediaServer(_PS_IMG_) . _PS_IMG_ . $logo;
+        } else {
+            $logo_path = _PS_IMG_ . $logo;
+        }
+
         $this->smarty->assign(array(
-            'logo_path' => Tools::getShopProtocol() . Tools::getMediaServer(_PS_IMG_) . _PS_IMG_ . $logo,
+            'logo_path' => $logo_path,
```

Comment:
Because of `img_ps_dir` you only need to do this:
```suggestion
            'logo_path' => _PS_IMG_ . $logo,
```